foundations/introduction/motivation_and_mindset.md

```diff
@@ -109,10 +109,10 @@ Procrastination will be your biggest enemy when trying to make progress.
 **Solution:** The [Pomodoro Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique) is a way of managing your time in order to stay focused. The idea is to set a timer for 25 minutes and to work on a task until the timer goes off. If you get distracted or interrupted during the 25 minutes, start the 25 minutes of work over again. Once you've successfully focused on work for 25 minutes, take a 5 minute break. When your break is over, repeat the 25 minutes of work and 5 minute break. After you've completed four 25 minute blocks of work, take a longer 15-30 minute break. 
 
 The Pomodoro technique is great for avoiding procrastination as it forces you to work without distractions. Since the work time only lasts 25 minutes before taking a break, it's not overwhelming, making it harder to rationalize procrastination.
-
+Pomofocus is a customizable pomodoro timer that works on desktop & mobile browser.
```

Comment:
This line is not needed because you have the same text on line 115

Reply:
@CouchofTomato removed the line, should be good to merge if you're good with the overall change here